packages/pg-connection-string/index.d.ts

```diff
@@ -7,9 +7,19 @@ export interface ConnectionOptions {
   port?: string | null
   database: string | null | undefined
   client_encoding?: string
-  ssl?: boolean | string
+  ssl?:
+    | string
+    | boolean
+    | {
+        cert: string
```

Comment:
This object is still missing `rejectUnauthorized: boolean` property as documented [here](https://github.com/brianc/node-postgres/tree/master/packages/pg-connection-string#usage). Connecting to a host with a self-signed ssl certificate and PGSSLMODE=required will have ssl set to true and will fail if `rejectUnauthorized` is not set to `false`. We ran into issues on Heroku with our application running with self signed certificates and had to pass `rejectUnauthorized: false` to resolve those issues.